### PR TITLE
namu.wiki - Ad (powerlink update)

### PR DESCRIPTION
**Pull request by seia-soto**
This commit adds support for updated namuwiki powerlink element and removes previous rules.

### CODE REVIEW

**Comment by seia-soto** [comment]
Please wait. There is invalid blocking rule and needs investigation.

**Comment by seia-soto** [comment]
The site structure has been changed. Will update the rule as soon as possible.

**Comment by seia-soto** [comment]
Updated the rule to fit new site structure.

File: filter.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ! Title: List-KR
 ! Description: List-KR
-! Version: 2022.322.11
+! Version: 2022.322.12
```
Comment (piquark6046) [inline filter.txt]:
```suggestion
! Version: 2022.322.13
```

**Review by piquark6046 — CHANGES_REQUESTED** [review]
Please update the version value.

**Comment by seia-soto** [comment]
Accepted the change and resolved the conflict.

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**